marketplaceItems/partyBall/entityResources/entityClientScripts/partyBall_client.js

```diff
@@ -20,6 +20,7 @@
     // *************************************
     // #region INIT
     
+    var log = Script.require('https://hifi-content.s3.amazonaws.com/milad/ROLC/d/ROLC_High-Fidelity/02_Organize/O_Projects/Repos/hifi-content/developerTools/sharedLibraries/easyLog/easyLog.js')
```

Comment:
please include this file in your project, then change to relative links for github

Reply:
oops sorry I missed that.  Should only be there while working.  fixing.